src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int diggerSize;
```

Comment:
Don't forget about access modifiers for your custom fields. It's a good practice to make fields private and provide public getters and setters for access.